### PR TITLE
Article Styling

### PR DESCRIPTION
**Pull request by peterp**
Removed "grey" when resolution is lower than 480

screenshot 2016-04-18 11 15 30
px.

screenshot 2016-04-18 11 15 35

### CODE REVIEW

**Comment by peterp** [comment]
article content:

screenshot 2016-04-18 14 40 17

article lists:

screenshot 2016-04-18 14 40 04

File: freebasics/templates/base.html [hunk 1]
```diff
@@ -35,56 +28,17 @@
...
-              }
-              return script;
-            }
-            // cut the mustard
-            if('querySelector' in document && 'localStorage' in window && 'addEventListener' in window) {
-              loadJS( "{% static 'js/main.js' %}" );
-            }
-            </script>
-
-            {% compress css file enhanced %}
```
Comment (codiebeulaine) [inline freebasics/templates/base.html:85]: @peterp what does this do? why are we taking it all out?

Comment (peterp) [inline freebasics/templates/base.html:85]: @codiebeulaine I think it's a bit of boilerplate JS that's meant to load JS asynchronously. It then loads a file called `main.js` which is completely empty.

The `enhanced` CSS file isn't used either.

Comment (codiebeulaine) [inline freebasics/templates/base.html]: @saeedmb8 all good to take this out?

Comment (codiebeulaine) [inline freebasics/templates/base.html:85]: @miltontony all good to take this out?

**Comment by codiebeulaine** [comment]
@peterp 👍

**Comment by miltontony** [comment]
:+1:

**Pull request merged**